index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end of google fonts -->
+    <!-- fontawesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- end of fontawesome -->
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css" />
+    <!-- end of style css -->
+    <title>portf</title>
+  </head>
+  <body>
+    <div class="header">
+      <div class="nav">
+        <div class="logo">
+          <a href="index.html"><h1 class="logo">LOGO</h1></a>
+        </div>
+
+        <div class="menu-bar">
+          <i class="fa-solid fa-bars"></i>
+        </div>
+      </div>
+      <div class="header-title">
+        <h1>
+          I'm Emmanuel.<br />
+          Grad to see you!
+        </h1>
+      </div>
+      <div class="header-about">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="header-contact">
+        <h4>let's connect</h4>
+      </div>
+      <div class="header-social-icons">
+        <a href="#"><i class="fa-brands fa-square-twitter"></i></a>
+        <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        <a href="#"><i class="fa-brands fa-medium"></i></a>
+      </div>
+    </div>
```

Comment:
- [ ] As I mentioned earlier please make sure to use the appropriate semantics for your elements 
    - Please place the code from lines `40-62` in a self-contained `section` element as this provides an appropriate separation of the different sections you'll have later on.
- [ ] After placing it in a section element add a `flexbox` styling to it as per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) 
- [ ] Also please be sure to add the appropriate background image as shown in the [Figma design 🎨 ](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62) 
      ![firefox_bucFyJSeTb](https://user-images.githubusercontent.com/104968236/227499946-f05bb4ab-5e1c-49d6-9517-83956d63cb9e.png)